Segment-Apptimize/Classes/SEGApptimizeIntegration.m

```diff
@@ -103,9 +103,4 @@ - (void)screen:(SEGScreenPayload *)payload
     [_apptimizeClass SEG_track:screenEvent attributes:payload.properties];
 }
 
-- (void)reset
```

Comment:
@honeybeep it's marked optional here: https://github.com/segmentio/analytics-ios/blob/master/Analytics/Classes/Integrations/SEGIntegration.h#L11